win/installer32.nsi.in

```diff
@@ -37,14 +37,14 @@ Section
 !include dll_list.nsh
 
         SetOutPath "$INSTDIR\platforms"
-        File platforms/qdirect2d.dll
-        File platforms/qminimal.dll
-        File platforms/qoffscreen.dll
-        File platforms/qwindows.dll
+        File platforms\qdirect2d.dll
```

Comment:
Looks like these reverse slash changes are going to break MXE build on Linux hosts. Maybe it's better to make a separate copy of NSIS script with Windows-style path delimiters?

Reply:
@curtishall Do we need support both MXE and MSYS2 build?

Reply:
@antonsviridenko at the moment I'm more interested in spending development time on adding features (HLS) then I am about different build environments.  msys2 seems to work well enough for now.

Reply:
yeah, but it is not reasonable to break things that work, and it is easy to make a copy of one file, so both build methods can work without issues